Sources/TUSKit/TUSClient.swift

```diff
@@ -200,9 +200,14 @@ public final class TUSClient: NSObject {
     /// - Returns: ANn id
     /// - Throws: TUSClientError
     @discardableResult
-    public func uploadFile(filePath: URL, uploadURL: URL? = nil, customHeaders: [String: String] = [:], context: [String: String]? = nil) throws -> UUID {
+    public func uploadFile(uploadId: Any?, filePath: URL, uploadURL: URL? = nil, customHeaders: [String: String] = [:], context: [String: String]? = nil) throws -> UUID {
```

Comment:
@Cantabar why not give it a type of UUID? instead of Any?

Reply:
That starts going down a rabbit hole that makes it so we have to create a struct to define the upload object. It would be good to do that eventually but I didn't want to delay this PR to work through all of those issues.